adapters/airnow.js

```diff
@@ -16,149 +19,116 @@ exports.fetchData = function (source, cb) {
   }
 
   // First fetch the stations list and then get the latest measurements
-  const file = 'Locations/monitoring_site_locations.dat';
-  const lineToObj = function (line) {
-    const convertCity = function (city) {
-      if (!city) {
-        return;
-      }
-
-      return city.split(',')[0].trim();
-    };
-
-    return {
-      aqsid: line[0],
-      coordinates: {
-        latitude: Number(line[8]),
-        longitude: Number(line[9])
-      },
-      country: line[12],
-      city: convertCity(line[16]) || line[20]
-    };
-  };
-  // Get the stations
-  getObjects(source.url, file, lineToObj, (err, stations) => {
-    if (err) {
-      return cb({message: 'Failure to load ftp data.'});
-    }
-
-    const niceUnit = function (unit) {
-      switch (unit) {
-        case 'UG/M3':
-          return 'µg/m³';
-        default:
-          return unit.toLowerCase();
-      }
-    };
-
-    const niceParameter = function (parameter) {
-      switch (parameter) {
-        case 'OZONE':
-          return 'o3';
-        case 'PM2.5':
-          return 'pm25';
-        default:
-          return parameter.toLowerCase();
-      }
-    };
-
-    // Get the measurements
-    // Filename should be the current time in UTC like '2016030616' and then
-    // get the previous hours measurements
-    const dateString = moment.utc().subtract(1, 'hours').format('YYYYMMDDHH');
-    const file = `HourlyData/${dateString}.dat`;
-    const lineToObj = function (line) {
+  getStream('Locations/monitoring_site_locations.dat')
+    .map(line => {
```

Comment:
Getting a `getStream(...).map is not a function` error for this adapter here.

Reply:
Will check